utils/useCheckMobileScreen.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react'
+
+const useCheckMobileScreen = () => {
+  const [width, setWidth] = useState(0)
+  const handleWindowSizeChange = () => {
```

Comment:
This is one way to check mobile, might be easier just to use the media query hook that comes with MUI. Check out how I [did it in orcasite](https://github.com/orcasound/orcasite/blob/f745019967ef1f474ea0af7ad8c8cabab74ace7e/ui/src/hooks/useIsMobile.ts)